### PR TITLE
docs: update reference to community templates

### LINKED ISSUE #178: Community Templates link broken
#### Overview of the Issue

In the file `docs/builders/qemu.mdx`, the link with with the text "Community Templates" is dead.

### PR DESCRIPTION
**Pull request by lbajolet-hashicorp**
The Community Templates link from the builder did not link to the community templates general documentation page, and instead was a broken link.

Closes #178

### CODE REVIEW

File: .web-docs/components/builder/qemu/README.md
Comment (nywilken) [inline .web-docs/components/builder/qemu/README.md:79]:
```suggestion
[Community Templates](https://developer.hashicorp.com/packer/docs/community-tools#templates)
```

File: docs/builders/qemu.mdx
Comment (nywilken) [inline docs/builders/qemu.mdx:90]:
```suggestion
[Community Templates](https://developer.hashicorp.com/packer/docs/community-tools#templates)
```

**Review by nywilken — APPROVED** [review]
These should point to the developer portal, as packer.io is reserved fro marketing.

**Comment by lbajolet-hashicorp** [comment]
Good catch! Updated this now, I'll wait until the tests go green and merge.

**Pull request merged**
